receiver/vcenterreceiver/metadata.yaml

```
@@ -7,10 +7,16 @@ status:
     alpha: [metrics]
   distributions: [contrib]
   codeowners:
-    active: [djaglowski, schmikei]
+    active: [djaglowski, schmikei, StefanKurek]
     seeking_new: true
 
-resource_attributes:
+resource_attributes:  
+  vcenter.datacenter.name:
+    description: The name of the vCenter datacenter.
+    enabled: false
+    type: string
+    warnings:
+      if_enabled_not_set: "this attribute will be enabled by default starting in release v0.100.0"
```

v0.100.0 is the next release at this point so v0.101.0 would be the soonest possible to enable it by default.
```suggestion
      if_enabled_not_set: "this attribute will be enabled by default starting in release v0.101.0"
```